dashboard/test/ui/features/signing_in.feature

```diff
@@ -35,6 +35,22 @@ Scenario: Student sign in from studio.code.org
   Then I wait to see ".user_menu"
   Then element ".user_menu span:first" has text "Hi Alice"
 
+Scenario: Student sign in from studio.code.org in the eu
+  Given I am on "http://studio.code.org/"
+  And I set the language cookie
+  And I create a student in the eu named "Alice"
+  And I sign out
```

Comment:
Why do you have to sign out here?

Reply:
I just copied this UI test, but it appears that the test then verifies that we can successfully sign back in using the just-created account.